### PR TITLE
Writing to dpm@Vienna

### PR DESCRIPTION
**Pull request by schoef**
updated signal samples in samples_13TeV_CSA14.py and samples_13TeV_CSA14v2.py
fileprefix to ease local running (was asked for by Maria)
extension of pybatch.py and batchmanager.py in order to write to Tier2 in Vienna

closed #176 because it is redundant.

### CODE REVIEW

**Comment by cbotta** [comment]
@schoef 
refering to how you added new samples in: CMGTools/TTHAnalysis/python/samples/samples_13TeV_CSA14.py
Can you please move the samples you just added together with the other signal samples we produced privately (## v2 but also available here to make running easier (they're compatible with v1 anyway)),
which are then included in the sub-category:
[mcPrivateSamplesCSA14v2]

I don't think a specific sub-category for ACDV samples is needed, expecially because in [mcPrivateSamplesCSA14v2] we have already your T5WW_2J_\* just above. You can maybe renominate them with T5qqqqWW.

Can you also do that (adding the recently produced signal samples by both groups) for python/samples/samples_13TeV_CSA14v2.py?

Thanks a lot

**Comment by schoef** [comment]
Done. Give it a look. This now also contains #176  which I will close.

File: CMGTools/TTHAnalysis/python/samples/samples_13TeV_CSA14.py [hunk 1]
```diff
@@ -318,9 +315,10 @@
...
-T5WW_2J_mGo1400_mCh315_mChi300 = kreator.makePrivateMCComponent('T5WW_2J_mGo1400_mCh315_mChi300','', ["/store/cmst3/group/susy/alobanov/MC/MiniAOD_v2/13TeV_T5qqqqWW_Gl_1400_LSP_300_Chi_315/13TeV_T5qqqqWW_Gl_1400_LSP_300_Chi_315_MiniAOD-v2.root"],0.0252977)
+T5qqqqWW_2J_mGo1200_mCh1000_mChi800 = kreator.makeMCComponentFromEOS('T5WW_2J_mGo1200_mCh1000_mChi800', '/T5Full-1200-1000-800-Decay-MGMMatch50/', '/store/cmst3/group/susy/gpetrucc/13TeV/MINIAODSIM/%s',".*root",0.0856418)
+T5qqqqWW_2J_mGo1500_mCh800_mChi100  = kreator.makeMCComponentFromEOS('T5WW_2J_mGo1500_mCh800_mChi100', '/T5Full-1500-800-100-Decay-MGMMatch50/', '/store/cmst3/group/susy/gpetrucc/13TeV/MINIAODSIM/%s',".*root",0.0141903)
+T5qqqqWW_2J_mGo1400_mCh315_mChi300 = kreator.makePrivateMCComponent('T5WW_2J_mGo1400_mCh315_mChi300','', ["/store/cmst3/group/susy/alobanov/MC/MiniAOD_v2/13TeV_T5qqqqWW_Gl_1400_LSP_300_Chi_315/13TeV_T5qqqqWW_Gl_1400_LSP_300_Chi_315_MiniAOD-v2.root"],0.0252977)
+T5qqqqWW_2J_mGo1400_mCh315_mChi325 = kreator.makeMCComponentFromEOS('T5qqqqWW_Gl_1400_LSP_100_Chi_325', '/13TeV_T5qqqqWW_Gl_1400_LSP_100_Chi_325/', '/store/cmst3/group/susy/alobanov/MC/MiniAOD_v2/%s', ".*root", 0.0252977)
```
Comment (gpetruc) [inline CMGTools/TTHAnalysis/python/samples/samples_13TeV_CSA14.py]: You should change also the name in the string passed as first argument of `makeMCComponentFromEOS` and not just the name of the object in python.  i.e. it must be `Xyz = kreator.makeSomething('Xyz', ...)`  and not  `Xyz = kreator.makeSomething('Abcd', ...)` 

Can you fix this in both files?

Sorry for taking this long to look at them

Giovanni

File: CMGTools/TTHAnalysis/python/samples/samples_13TeV_CSA14v2.py [hunk 2]
```diff
@@ -119,11 +119,10 @@
 ## MORE private samples on EOS
 ### PU40 bx50ns
 #### Signal samples
-T5WW_2J_mGo1200_mCh1000_mChi800 = kreator.makeMCComponentFromEOS('T5WW_2J_mGo1200_mCh1000_mChi800', '/T5Full-1200-1000-800-Decay-MGMMatch50/', '/store/cmst3/group/susy/gpetrucc/13TeV/MINIAODSIM/%s',".*root",0.0856418)
-T5WW_2J_mGo1500_mCh800_mChi100  = kreator.makeMCComponentFromEOS('T5WW_2J_mGo1500_mCh800_mChi100', '/T5Full-1500-800-100-Decay-MGMMatch50/', '/store/cmst3/group/susy/gpetrucc/13TeV/MINIAODSIM/%s',".*root",0.0141903)
-T5WW_2J_mGo1400_mCh315_mChi300 = kreator.makePrivateMCComponent('T5WW_2J_mGo1400_mCh315_mChi300','', ["/store/cmst3/group/susy/alobanov/MC/MiniAOD_v2/13TeV_T5qqqqWW_Gl_1400_LSP_300_Chi_315/13TeV_T5qqqqWW_Gl_1400_LSP_300_Chi_315_MiniAOD-v2.root"],0.0252977)
-
-T1tttt_2J_mGo1000_mStop300_mCh285_mChi280 = kreator.makeMCComponentFromEOS('T1tttt_2J_mGo1000_mStop300_mCh285_mChi280', '/T1tttt_2J_mGo1000_mStop300_mCh285_mChi280_pythia8-23bodydec/', '/store/cmst3/group/susy/gpetrucc/13TeV/MINIAODSIM/%s',".*root",0.325388)
```
Comment (gpetruc) [inline CMGTools/TTHAnalysis/python/samples/samples_13TeV_CSA14v2.py:126]: Why did you remove the `T1tttt_2J_mGo1000_mStop300_mCh285_mChi280` sample? was it duplicated somewhere?

**Comment by schoef** [comment]
@gpetruc 
Thanks for spotting. Both things were not on purpose.

**Comment by gpetruc** [comment]
Unless I'm mistaken, you're still removing `T1tttt_2J_mGo1000_mStop300_mCh285_mChi280`  from `samples_13TeV_CSA14v2.py`. Can you fix that as well?

**Comment by schoef** [comment]
@gpetruc 
Done.
